src/components/PeopleList/PeopleList.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect, useState } from 'react';
+import { getPeople } from '../../api/api';
+import { PeopleTable } from '../PeopleTable';
+
+export const PeopleList: React.FC = () => {
+  const [peopleList, setPeopleList] = useState<People[]>([]);
+
+  async function loader() {
+    const peopleFromServer = await getPeople();
+
+    if (Array.isArray(peopleFromServer)) {
+      const resultPeopleList = peopleFromServer
+        .map((people, _, peoplesList) => {
```

Comment:
use some other names, there's no such word as `peoples`
```suggestion
        .map((people, _, reference) => {
```